### ISSUE TITLE
Show times on events in day and week view

### ISSUE THREAD

**Issue by pl4nty** [anucssa/anutimetable#79]
Requested by Talon on discord

I don't think this should be the default, to avoid clutter (particularly in the mobile view). Instead, we could surface an option in the FAB.
Just need to update the `displayEventTime` calendar prop

**Comment by OliverBalfour** [comment]
Updating the prop to `displayEventTime={true}` doesn't work, maybe because we've overridden the event contents view

**Comment by pl4nty** [comment]
Yeah that sounds right. I wonder if we can hook the function FC uses, [buildSegTimeText](https://github.com/fullcalendar/fullcalendar/blob/53de3e4882e2efaef6ec9c8285fa6d6777c688b0/packages/common/src/component/event-rendering.ts#L234)?
